Environment/NoiseLevelObserved/doc/spec.md

```diff
@@ -60,33 +59,24 @@ It allows to convey, roughly, information about the precision of the measurement
     + Attribute type: [Text](https://schema.org/Text)
     + Allowed values: one of (`"0"`, `"1"`, `"2"`)
     + Optional
+    
```

Comment:
if it's always a number, why not using number?

Reply:
because it is not semantically a number, but a class, so I prefer to keep it text. In addition we can model better the allowed values by enumeration